lib/perl/Genome/Role/Comparable/Command/DiffBlessed.pm

```diff
@@ -11,6 +11,14 @@ role Genome::Role::Comparable::Command::DiffBlessed {
     requires => ['new_object'],
 };
 
+sub help_brief : Overrides(Genome::Role::Comparable::Command::Diff) {
```

Comment:
This is an artifact of me trying to let this role have the 'diff' role. The methods from the diff role were not conferred to the diff-blessed role, and I abandoned this fix. I will remove it, and have additionally fixed the help to not refer to processes, but objects.